ui/shared/components/form/ReduxFormWrapper.jsx

```diff
@@ -147,6 +147,9 @@ class ReduxFormWrapper extends React.Component {
     /* React child component class. Mutually exclusive with fields */
     children: PropTypes.node,
 
+    /* Call if submit succeeded */
+    closeInlineModal: PropTypes.func,
```

Comment:
I would name this something more generic, like `onSubmitSucceeded`, so that you can pass any function you want to call after a successful submission